testing/smoke-test/vite.config.ts

```diff
@@ -0,0 +1,31 @@
+import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vite';
+import dynamicImport from 'vite-plugin-dynamic-import';
+
+// https://vite.dev/config/
+export default defineConfig({
+  plugins: [
+    react(),
+    /*
+     * This plugin is required to resolve the charts-core module as they are dynamically imported
+     * when the library is loaded.
+     * See the dynamic loading code here: https://github.com/ionic-team/stencil/blob/main/src/client/client-load-module.ts#L32
+     * Unfortunately, we are using an older version of the stencil library which doesn't use the syntax vite supports to
+     * detect the dynamic imports automatically, so we need to use this plugin to explicitly tell vite to include the
+     * charts-core module otherwise their code will not be included in the build.
+     */
+    dynamicImport({
```

Comment:
This is only required in the case of adding something like the bar-chart / status-timeline to the dashboard right?